src/MySql/custom/New-AzMySqlFlexibleServer.ps1

```diff
@@ -147,60 +183,418 @@ function New-AzMySqlFlexibleServer {
 
     process {
         try {
-            if ($PSBoundParameters.ContainsKey('Name')) {
-              $PSBoundParameters.ServerName = $PSBoundParameters['Name']
-              $null = $PSBoundParameters.Remove('Name')
+            If (!$PSBoundParameters.ContainsKey('Location')) {
+                $PSBoundParameters.Location = 'westus2'
+            }
+
+            If (!$PSBoundParameters.ContainsKey('AdministratorLoginPassword')) {
+                $Password = Get-GeneratePassword
+                $PSBoundParameters.AdministratorLoginPassword = $Password | ConvertTo-SecureString -AsPlainText -Force
+            }
+            $PSBoundParameters.AdministratorLoginPassword = . "$PSScriptRoot/../utils/Unprotect-SecureString.ps1" $PSBoundParameters['AdministratorLoginPassword']
+
+            If (!(Get-Module -ListAvailable -Name Az.Resources)) { Throw 'Please install Az.Resources module by entering "Install-Module -Name Az.Resources"'}
+            Else { Import-Module -Name Az.Resources }
```

Comment:
You can Import-Module -Name Az.Resources directly. A error will ocurr if the module is not installed

Reply:
Will do!